company_today/models/res_company.py

```diff
@@ -0,0 +1,20 @@
+# SPDX-FileCopyrightText: 2022 Coop IT Easy SCRLfs
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+from odoo import api, fields, models
+
+
+class Company(models.Model):
+    _inherit = "res.company"
+
+    today = fields.Date(
+        string="Today's Date",
+        default=lambda _: fields.Date.today(),
+        readonly=True,
+    )
+
+    @api.model
+    def cron_update_today(self):
+        companies = self.search([])
+        companies.write({"today": fields.Date.today()})
```

Comment:
does it work to update a read-only field? shouldn’t it be a computed field that depends on nothing, and whose compute function is called by the cron? if it works like this, great!

Reply:
Yes, `read_only=False` prevents the user to edit it but you can write on it.

Reply:
good to know, thanks!